### ISSUE TITLE
preset value being ignored

### ISSUE THREAD

**Issue by cxjr** [checkmarx-ts/checkmarx-cxflow-github-action#9]
Seems like the preset value is being ignored both as a param value and as an input.  Seems like the values make it to the docker container properly but CxFlow ends up using Checkmarx Default instead.

Tried both with named values and numeric numbers.

test repo is here pointing at a POC instance.
https://github.com/cxjr/Cx_actions_sast_SCA/blob/main/.github/workflows/checkmarx-scan-pr.yml

**Comment by kmcdon83** [comment]
@cxjr we have logic to only apply a new preset if the project is new.  I just tried your scenario and was able to get Checkmarx Express once I deleted my project.

Also, please check your params input value.  You have a reference to --checkmarx.scan-preset which will again override the value.

**Comment by kmcdon83** [comment]
First, we had issues created because we were replacing the preset each time, now we get the reverse

**Comment by cxjr** [comment]
That makes total sense to constrain a new preset like that from an external source.  I didn't realize their was logic for that though.
I do think their is an issue though, I tested the following:
1.) As a regular input Checkmarx Express on a brand new SAST project - Checkmarx Default was used.
2.) As a param on a brand new project as Checkmarx Express, and it seems like the space in the preset name throws it off.  'Checkmarx Express' is recognized just as Checkmarx and nothing is found.  I tried with single and double quotes and neither worked. 

--checkmarx.scan-preset=Checkmarx Express
